README.md

```diff
@@ -201,31 +201,60 @@ download and building of hypre, METIS and MFEM.
 
 #### Sedov blast
 
-The main problem of interest for Laghos is the Sedov blast wave (`-p 1`) with
-partial assembly option (`-pa`).
+The 3D Sedov blast wave problem can be runned with `-p 1`.
 
-Some sample runs in 2D and 3D respectively are:
+A sample run of the offline stage is:
```

Comment:
@siuwuncheung Should the ROM updates only be in `rom/README.md`, or is there a reason to update this main README as well?

Reply:
@dylan-copeland Thanks for your reminder. I have edited the wrong file.